x-pack/plugins/maps/public/classes/styles/vector/style_util.ts

```diff
@@ -19,7 +18,7 @@ export const OTHER_CATEGORY_LABEL = i18n.translate(
   }
 );
 
-export const OTHER_CATEGORY_DEFAULT_COLOR = euiThemeVars.euiColorLightShade;
+export const OTHER_CATEGORY_DEFAULT_COLOR_TOKEN = 'textSubdued';
```

Comment:
'textSubdued' is not going to work. This value is passed to maplibre for rendering and must be a hex value. For example, this is what a map looks like with this value 

<img width="1242" alt="Screenshot 2024-12-18 at 7 54 00 AM" src="https://github.com/user-attachments/assets/8954d9b3-ba4b-437a-88f1-0093a50ee54f" />

When instead, it should look like this 
<img width="1242" alt="Screenshot 2024-12-18 at 7 55 18 AM" src="https://github.com/user-attachments/assets/1797d2e2-6a2c-45ca-80c4-28f7d54c4b22" />